packages/ic-asset-certification/src/asset_router.rs

```diff
@@ -269,6 +269,9 @@ impl<'content> AssetRouter<'content> {
     ///
     /// If no configuration matches an individual asset, the asset will be
     /// served and certified as-is, without headers.
+    ///
+    /// After performing this operation, the canister's certified variable will need to be updated
```

Comment:
nit: I'd rephrase to something more direct, e.g. "After this operation one must set canister's certified data (`ic_cdk::set_certified data()`) to the new [root hash](AssetRouter::root_hash) of the tree." 
(also for the other functions below)